src/components/views/transaction/Transaction.view.tsx

```diff
@@ -111,7 +112,30 @@ const TransactionView = () => {
 			}
 		};
 
+		const socket = new WebSocket('ws://localhost:4000');
```

Comment:
@Meriem-BM I think you should read the address from config instead of hard coded URL (it's not localhost:4000 for staging and production)

Reply:
Thanks for catching that. literally forgot it.